presto-docs/src/main/sphinx/security/file-based-system-access-control.rst

```diff
@@ -0,0 +1,93 @@
+================================
+File Based System Access Control
+================================
+
+Presto offers a file based ``SystemAccessControl`` plugin that supports
+specifying access control rules using JSON in a file. In order to enable the
+plugin, you must add an ``etc/access-control.properties`` file containing two
+required properties, ``access-control.name`` which must be equal to ``file``,
+and ``security.config-file`` which must be set to the location of the JSON
+file. For example, if a JSON file named ``rules.json`` resides in ``etc``,
+add an ``etc/access-control.properties`` with the following contents:
+
+.. code-block:: none
+
+   access-control.name=file
+   security.config-file=etc/rules.json
+
+The ``rules.json`` file should consist of a list of access control rules.
+These rules should be specified in the order in which they are intended to be
+matched. The user is granted the privileges from the first matching rule. All
+regular expressions default to ``.*`` if not specified.
+
+At present, only the catalog access control rules are supported in this
+plugin. Other access control rules such as those for schemas or tables or
+session properties are not yet supported.
+
+Catalog Rules
+-------------
+
+These rules govern the accesses allowed on specified catalogs for the specified
+users. The user is granted the access to a catalog based on the first matching
+rule. In case no rule matches, the user is denied the access to the catalog.
+Thus, in absence of any rules, access to all catalogs is denied for all users
+when this plugin is enabled. Each rule is composed of the following fields:
+
+* ``user`` (optional): regex to match against user name.
+
+* ``catalog`` (optional): regex to match against catalog name.
+
+* ``allowed`` (required): boolean indicating whether a user has access to the catalog
+
+Consider a use case in which the only access control restriction you want is
```

Comment:
suggest:
For example, if you want to disallow access to the ``mysql`` catalog to the user ``alice`` but allow all other access, you can use the following rules: